PWGHF/TableProducer/trackIndexSkimCreator.cxx

```diff
@@ -3234,7 +3235,11 @@ struct HfTrackIndexSkimCreatorLfCascades {
     // Selection criteria
     // selections have been set to run2 lambda dedicated cuts
     // selections for cascade have been set to the loosest value between xi and omega
-    // a tolerance has been added to be more conservative
+    // a tolerance has been added to be more conservative  ptMinOmegaczeroToOmegaKaLfCasc ptMinXicZeroOmegacZeroToXiPiLfCasc
```

Comment:
Why did you add `ptMinOmegaczeroToOmegaKaLfCasc ptMinXicZeroOmegacZeroToXiPiLfCasc` here?

Reply:
Dear expert, thank you for comment! I added these four configurable because Fabrizio suggested doing this. We think it's better to cut pT for OmegaczeroToOmegaKa, XicZeroOmegacZeroToXiPi, OmegacZeroToOmegaPi and XicplusToXiPiPi respectively.

Reply:
@vkucera this is for the production of derived data in Pb-Pb, for which it is unlikely that we can perform measurements to pT=0 and the size is drastically reduced if we skip the low pT

Reply:
@FDUEnrich @fgrosa I was referring to that very line with the comment.